docs/source/sysadmin_guide/configuring_permissions.rst

```diff
@@ -84,106 +121,114 @@ transactions. Each role equals a policy name that corresponds to a policy
 
-default:
+``default``:
   When evaluating role permissions, if the role has not been set, the default
-  policy will be used. The policy can be changed to meet the network's
+  policy is used. The policy can be changed to meet the network's
   requirements after initial start-up by submitting a new policy with the name
   default. If the default policy has not been explicitly set, the default
-  is “PERMIT_KEY \*”.
+  is ``PERMIT_KEY \*``.
 
-transactor:
+``transactor``:
   The top level role for controlling who can sign transactions and batches on
-  the system will be transactor. This role shall be used when the allowed
+  the system. This role is used when the allowed
   transactors for transactions and batches are the same. Any transactor whose
-  public key is in the policy will be allowed to sign transactions and batches,
+  public key is in the policy is allowed to sign transactions and batches,
   unless a more specific sub-role disallows their public key.
 
-transactor.transaction_signer:
+``transactor.transaction_signer``:
   If a transaction is received that is signed by a transactor who is not
   permitted by the policy, the batch containing the transaction will be dropped.
 
-transactor.transaction_signer.{tp_name}:
+``transactor.transaction_signer.{tp_name}``:
   If a transaction is received for a specific transaction family that is signed
   by a transactor who is not permitted by the policy, the batch containing the
-  transaction will be dropped.
+  transaction will be dropped. Replace {tp_name} with a transaction family
+  name (see :doc:`../transaction_family_specifications`.
 
-transactor.batch_signer:
+``transactor.batch_signer``:
   If a batch is received that is signed by a transactor who is not permitted by
   the policy, that batch will be dropped.
 
 Validator Key Permissioning
 ===========================
 
-Overview
---------
-
-One of the permissioning requirements is that the validator network be able to
+Sawtooth allows the validator network to
 limit the nodes that are able to connect to it. The permissioning rules
-determine the roles a connection is able to play on the network. The roles
+determine the roles a connection is able to play on the network. These roles
 control the types of messages that can be sent and received over a given
-connection. The entities acting in the different roles will be referred to as
-requesters below.
+connection. In this section, the entities acting in the different roles are
+referred to as "requesters".
 
 Validators are able to determine whether messages delivered to them should
 be handled or dropped based on a set of role and identities stored within the
-Identity namespace. Each requester will be identified by the public key derived
-from their identity signing key. Permission verifiers examine incoming
+Identity namespace. Each requester is identified by the public key derived
+from its Identity signing key. Permission verifiers examine incoming
```

Comment:
Did you capitalize this because of the Identity transaction processor? This is the public key used by the validator.

Reply:
Would it be correct to change this phrase to "Each requester is identified by its public key?"